.github/workflows/stale.yaml

```diff
@@ -14,12 +14,18 @@ jobs:
     - uses: actions/stale@v4
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-pr-stale: 14
+        days-before-pr-close: 30
+        stale-pr-label: 'lifecycle/stale'
         stale-pr-message: >
           Marking this PR stale since there has been no activity for 14
           days. It will be closed if there is no activity for another
           30 days.
-        stale-pr-label: 'lifecycle/stale'
-        days-before-stale: 14
-        days-before-close: 30
-        days-before-issue-stale: -1 # disable marking issues as stale
-        days-before-issue-close: -1 # disable closing stale issues
+        days-before-issue-stale: 60
+        days-before-issue-close: 30
+        stale-issue-label: 'lifecycle/stale'
+        exempt-issue-labels: 'help wanted,good first issue,priority/important-soon'
+        stale-issue-message: >
+          Marking this Issue stale since there has been no activity for 60
+          days. It will be closed if there is no activity for another
+          30 days.
```

Comment:
I'm thinking it'd be helpful to expand this message a bit, to be more like what upstream k8s projects use (e.g. https://github.com/kubernetes-sigs/gateway-api/issues/1180#issuecomment-1250439469):

- explain why we have the stale bot
- explain what the stale bot does (this is covered already)
- explain what actions the user can take (comment, close, contribute)

I think the extra bit of context is a little more friendly and gives the user specific steps they can take if they want.